### PR TITLE
Changes to global input

### PR DESCRIPTION
**Pull request by oksuzian**

### CODE REVIEW

File: ProjPy/mdc2020_primary.ini [hunk 1]
```diff
@@ -34,31 +34,31 @@ test_param_overrides = []
...
+[dependencies primary_generation]
+campaign_stage_1 = primary_generation_par
 file_pattern_1 = %.tar
 
-[job_type mu2epro_primary_jobtype_ai]
-launch_script = fife_launch
-parameters = [["-c ", "/cvmfs/mu2e.opensciencegrid.org/Musings/SimJob/MDC2020ai/Production/CampaignConfig/mdc2020_primary.cfg"]]
-output_file_patterns = %.art
-recoveries = [["process_status", [["-Osubmit.memory=", "4GB"], ["-Oglobal.dataset=", "%(dataset)s"] ]], ["process_status", [["-Osubmit.memory=", "8GB"], ["-Oglobal.dataset=", "%(dataset)s"]]]]
+#[job_type mu2epro_primary_jobtype_ai]
```
Comment (brownd1978) [inline ProjPy/mdc2020_primary.ini:55]: Is there a reason these lines below are commented out and not removed?

File: data/primary.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [
     {
-	"release_v_dts": "ai",
+	"release_v_o": "ai",
```
Comment (brownd1978) [inline data/primary.json:3]: what does v_o stand for?

File: data/primary.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
...
 	"stopped_type": "Muminus",
 	"release_v_stops": "p",
 	"primary_name": "DIOtail_75",
```
Comment (brownd1978) [inline data/primary.json:6]: start_mom is missing from this stanza, is that a problem?

**Review by brownd1978 — COMMENTED** [review]
Just a few questions.

File: data/primary.json (hunk 2 above)
Comment (oksuzian) [reply data/primary.json:3]: v_o = version_out
v_i = version_in

File: data/primary.json (hunk 3 above)
Comment (oksuzian) [reply data/primary.json:6]: Are you referring to stop_mom?
stop_mom is not used as the variable at the moment.

start_mom is used though.

File: ProjPy/mdc2020_primary.ini (hunk 1 above)
Comment (oksuzian) [reply ProjPy/mdc2020_primary.ini]: Removed

File: CampaignConfig/mdc2020_primary.cfg [hunk 4]
```diff
@@ -37,18 +31,22 @@ executable_2.arg_9 = --start %(start_mom)s
...
+executable_4.name = \\\\\\\$CONDOR_DIR_INPUT/run_JITfcl.py
+
 job_output_1.dest = %(logdir_bck)s/%(stage_name)s/%(desc)s/tbz/
 job_output_2.add_to_dataset = dts.%(submitter)s.%(stage_name)s.%(desc)s.art
 job_output_2.dest = %(outdir_dts_tape)s/%(stage_name)s/%(desc)s/art
 global.artRoot_dataset = dts.%(submitter)s.%(stage_name)s.%(desc)s.art
 
 executable.name = true
-executable_4.name = run_JITfcl.py
+#executable_4.name = run_JITfcl.py
```
Comment (brownd1978) [inline CampaignConfig/mdc2020_primary.cfg:49]: can this line be removed?

File: CampaignConfig/mdc2020_prolog.cfg [hunk 5]
```diff
@@ -19,7 +19,10 @@ outdir_tar      = /pnfs/mu2e/persistent/datasets/phy-etc/cnf/mu2e/
 logdir_bck_tape = /pnfs/mu2e/persistent/datasets/phy-etc/bck/mu2e/
 outdir_sim_nersc = /pnfs/mu2e/tape/phy-sim/sim/mu2e/
 
-simjob_release = /cvmfs/mu2e.opensciencegrid.org/Musings/SimJob/%(release)s%(release_v_o)s/setup.sh
+bfield = "Offline/Mu2eG4/geom/bfgeom_no_tsu_ps_v01.txt"
```
Comment (brownd1978) [inline CampaignConfig/mdc2020_prolog.cfg:22]: this bfield is correct for stopped muon primaries, but not for beam or cosmic primaries. Should this go in mdc2020_primary instead?

Comment (oksuzian) [reply CampaignConfig/mdc2020_prolog.cfg:22]: I'm confused, but that's the only file that I see in Production:

$ grep -r bfield /cvmfs/mu2e.opensciencegrid.org/Musings/SimJob/MDC2020aj/Production/CampaignConfig/*
/cvmfs/mu2e.opensciencegrid.org/Musings/SimJob/MDC2020aj/Production/CampaignConfig/fclless/mdc2024_primary.cfg:bfield = "Offline/Mu2eG4/geom/bfgeom_no_tsu_ps_v01.txt"
/cvmfs/mu2e.opensciencegrid.org/Musings/SimJob/MDC2020aj/Production/CampaignConfig/fclless/mdc2024_primary.cfg:executable_2.arg_7 = --field %(bfield)s
/cvmfs/mu2e.opensciencegrid.org/Musings/SimJob/MDC2020aj/Production/CampaignConfig/mdc2020_corsikamix.cfg:bfield = "Offline/Mu2eG4/geom/bfgeom_no_tsu_ps_v01.txt"
/cvmfs/mu2e.opensciencegrid.org/Musings/SimJob/MDC2020aj/Production/CampaignConfig/mdc2020_corsikamix.cfg:executable_2.arg_8 = --field %(bfield)s
/cvmfs/mu2e.opensciencegrid.org/Musings/SimJob/MDC2020aj/Production/CampaignConfig/mdc2020_corsikamix.cfg:executable_2.arg_8 = --field %(bfield)s
/cvmfs/mu2e.opensciencegrid.org/Musings/SimJob/MDC2020aj/Production/CampaignConfig/mdc2020_crymix.cfg:bfield = "Offline/Mu2eG4/geom/bfgeom_no_tsu_ps_v01.txt"
/cvmfs/mu2e.opensciencegrid.org/Musings/SimJob/MDC2020aj/Production/CampaignConfig/mdc2020_crymix.cfg:executable_2.arg_8 = --field %(bfield)s
/cvmfs/mu2e.opensciencegrid.org/Musings/SimJob/MDC2020aj/Production/CampaignConfig/mdc2020_crymix.cfg:executable_2.arg_8 = --field %(bfield)s
/cvmfs/mu2e.opensciencegrid.org/Musings/SimJob/MDC2020aj/Production/CampaignConfig/mdc2020_digireco.cfg:bfield = "Offline/Mu2eG4/geom/bfgeom_no_tsu_ps_v01.txt"
/cvmfs/mu2e.opensciencegrid.org/Musings/SimJob/MDC2020aj/Production/CampaignConfig/mdc2020_digireco.cfg:executable_2.arg_8 = --field %(bfield)s
/cvmfs/mu2e.opensciencegrid.org/Musings/SimJob/MDC2020aj/Production/CampaignConfig/mdc2020_digireco.cfg:executable_2.arg_8 = --field %(bfield)s
/cvmfs/mu2e.opensciencegrid.org/Musings/SimJob/MDC2020aj/Production/CampaignConfig/mdc2020_primary.cfg:bfield = "Offline/Mu2eG4/geom/bfgeom_no_tsu_ps_v01.txt"
/cvmfs/mu2e.opensciencegrid.org/Musings/SimJob/MDC2020aj/Production/CampaignConfig/mdc2020_primary.cfg:executable_2.arg_7 = --field %(bfield)s

which is identical to the following in gen_Digitize: 
FIELD="Offline/Mu2eG4/geom/bfgeom_no_tsu_ps_v01.txt" #optional (for changing field map)

File: CampaignConfig/mdc2020_primary.cfg (hunk 4 above)
Comment (oksuzian) [reply CampaignConfig/mdc2020_primary.cfg]: Fixed

File: CampaignConfig/mdc2020_prolog.cfg (hunk 5 above)
Comment (brownd1978) [reply CampaignConfig/mdc2020_prolog.cfg:22]: mdc2020_beam.cfg uses the prolog default: Offline/Mu2eG4/geom/bfgeom_v01.txt
which is the full field.
standardServices.fcl is problematic for exactly this reason, it makes it nearly impossible to trace where things are coming from.

Comment (brownd1978) [reply CampaignConfig/mdc2020_prolog.cfg:22]: so my read is mdc2020_prolog.cfg would overwrite that with the wrong field (no_tsu_ps), is that right?

Comment (oksuzian) [reply CampaignConfig/mdc2020_prolog.cfg:22]: I don't understand this.
Are you talking about bfield variable specifically or something else?
Do you think we should chat?

File: CampaignConfig/mdc2020_primary.cfg [hunk 6]
```diff
@@ -1,32 +1,22 @@
...
+#release_v_stops = p
+#release_v_o = ag
+#stopped_type = Muminus
+#primary_name = NoPrimary
+#run_number = 1210
+#num_events = 2000
+#num_jobs = 250
+#start_mom = 0
+#end_mom = 110
+#bfield = "Offline/Mu2eG4/geom/bfgeom_no_tsu_ps_v01.txt"
```
Comment (brownd1978) [inline CampaignConfig/mdc2020_primary.cfg:11]: do we need these commented fields?

File: CampaignConfig/mdc2020_prolog.cfg (hunk 5 above)
Comment (brownd1978) [reply CampaignConfig/mdc2020_prolog.cfg:22]: The resolution is to add a comment stating that the bfield poms variable isn't used in beam campaigns.

**Review by brownd1978 — APPROVED** [review]

**Pull request merged**